### ISSUE TITLE
Exception thrown initializing audio on Ubuntu

### ISSUE THREAD

**Issue by GoogleCodeExporter** [cutelyaware/magiccube4d#114]
```
What steps will reproduce the problem?
1. Starting the program
2. Hovering over any cube of the tesseract

What is the expected output? What do you see instead?

Terminal's output is pasted at the end.


What version of the product are you using? On what operating system?

Magic Cube 4D: ver 4.0.185
Operating System: Ubuntu 14.04 x86_64


Please provide any additional information below.

Exception thrown has been attached as a text file.
All exceptions were thrown on one hover.
```

Original issue reported on code.google.com by `amoghda...@gmail.com` on 20 Jan 2015 at 7:32

Attachments:
* [Exception](https://storage.googleapis.com/google-code-attachments/magiccube4d/issue-114/comment-0/Exception)

**Comment by megaminxwin** [comment]
This is still an issue. I'm using OpenJDK 8 on Ubuntu, and nothing works at all.

**Comment by cutelyaware** [comment]
Can you send me your email address? I've got a patched version that I'd like you to test. It will disable the audio features in your situation but I hope it will otherwise work for you. Send to melinda at superliminal.com. Thanks!

**Comment by megaminxwin** [comment]
Done done. I didn't actually have audio, that didn't work either, but when I used it on other platforms I disabled it anyway. No harm done there.

**Comment by cutelyaware** [comment]
Resolved with commit 74d22d80ad719214dc03e00229933b081f35f27b. Thanks Tamara!
